tests/scripts/thread-cert/run_cert_suite.py

```diff
@@ -151,48 +151,48 @@ def release(self, port_offset: int):
         self._pool.put_nowait(port_offset)
 
 
+def print_summary(scripts: List[str], script_successes: Dict[str, List[int]], script_failures: Dict[str, List[int]]):
+    print("---------------------------------------")
+    print("Summary")
+    print("---------------------------------------")
+    for script in scripts:
+        success_count = len(script_successes[script])
+        failure_count = len(script_failures[script])
+        color = _COLOR_PASS if failure_count == 0 else _COLOR_FAIL
+        message = f'{color}PASS {success_count} FAIL {failure_count}{_COLOR_NONE} {script}'
+        if failure_count > 0:
+            message += f' {_COLOR_FAIL}Failed iterations: {script_failures[script]}{_COLOR_NONE}'
+        print(message)
+
+
 def run_tests(scripts: List[str], multiply: int = 1, run_directory: str = None):
-    script_fail_count = Counter()
-    script_succ_count = Counter()
+    scripts = list(set(scripts))
```

Comment:
Are we trying to do a sort here? Is it required?

Reply:
I'm not attempting to sort them but to deduplicate them. 

This is to avoid the situation where duplicate test scripts are provided as arguments. Without this step, the iterations will be counted multiple times.

Reply:
Got it!